gcg/entrypoint.py

```diff
@@ -143,6 +143,13 @@ def parse_args(argv):
         help="Verbosity level. Specify twice for debug logs as well.",
         action="count", default=0
     )
+    parser.add_argument(
+        '-P', '--prefix',
```

Comment:
How would you feel about making it --tag-prefix?
It's a little bit longer yet also more descriptive and specific (tag prefix? filename prefix? )

Reply:
Shure. Just a moment ...